apps/base-docs/docs/connecting-to-the-blockchain/configuring-providers-sbs.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Overview of Providers
```

Comment:
Should this have a different title than the other page? Right now they are identical.